assets/constants.ts

```diff
@@ -19,6 +19,7 @@
 
 export default {
   // Whether to allow custom event reporting to Google Analytics.
+  // Mock gtag library is used when disabled.
```

Comment:
Does this actually use a "mock gtag library", or should "library" be "function" instead?

Reply:
changed